training/general/src/environments/sklearn-1.1/context/conda_dependencies.yaml

```diff
@@ -19,4 +20,5 @@ dependencies:
   - azureml-mlflow=={{latest-pypi-version}}
   - azureml-telemetry=={{latest-pypi-version}}
   - scikit-learn~=1.1.0
+  - joblib~=1.2.0
```

Comment:
isn't scikit-learn bringing joblib?

Reply:
yes but joblib has a vulnerability. So we're adding an upgraded version